gazebo_plugins/src/gazebo_ros_force.cpp

```diff
@@ -72,6 +75,22 @@ void GazeboRosForce::Load(gazebo::physics::ModelPtr model, sdf::ElementPtr sdf)
     return;
   }
 
+  // Frame 
+  if (!sdf->HasElement("force_frame")) {
+    RCLCPP_ERROR(logger, "Force plugin missing <force_frame>, cannot proceed");
```

Comment:
This would break backwards compatibility with the current plugin. We should keep the current behaviour if `force_frame` is not defined, i.e. world.